packages/react-router-dom/__tests__/data-browser-router-test.tsx

```diff
@@ -4,6 +4,7 @@
 
 import { JSDOM } from "jsdom";
 import * as React from "react";
+import { act } from "react-test-renderer";
 import {
   render,
```

Comment:
You can get this from `@testing-library/react` instead. It forwards:

```suggestion
import {
  act,
  render,
```